rmgpy/cantherm/molepro.py

```diff
@@ -79,12 +274,47 @@ def loadEnergy(self,frequencyScaleFactor=1.):
                     or 'CCSD(T)-F12b total energy  ' in line):
                     E0=float(line.split()[-1])
                     break
+                if 'Electronic Energy at 0' in line:
+                    E0=float(line.split()[-2])
+                    break
                 line=f.readline()
-        
+
         f.close()
         
         #multiply E0 by correct constants
         if E0 is not None:
             E0 = E0 * constants.E_h * constants.Na
             return E0
         else: raise Exception('Unable to find energy in MolePro log file.')
+
+    def loadZeroPointEnergy(self):
+        """
+        Load the unscaled zero-point energy in J/mol from a MolPro log file.
+        """
+
+        modes = []
+        ZPE = None
+        spinMultiplicity = 1
+
+        f = open(self.path, 'r')
+        line = f.readline()
+        while line != '':
+
+            # Do NOT read the ZPE from the "E(ZPE)=" line, as this is the scaled version!
+            # We will read in the unscaled ZPE and later multiply the scaling factor
+            # from the input file
+
+            if 'Electronic Energy at 0 [K]:' in line:
```

Comment:
My test does not have an `Electronic Energy at 0 [K]` line, but rather ends with:
```
 UCCSD(T)-F12A/cc-pVTZ-F12 energy=   -547.974475211996

   UCCSD(T)-F12A        HF-SCF  
   -547.97447521   -547.20094715
```

Reply:
My fault, I didn't add the thermo keyword to the frequencies block in the molpro input file.
The above comment can be ignored, however could you add a note to the user in the Exception below to make sure the `thermo` option was added to the `{frequencies}` block?

Reply:
Ok I'll work on these tonight. You are correct that you need the thermo keyword...
 {frequencies
 thermo
 print,thermo}
I'll makes sure that is clear from the debug note.